### PR TITLE
Downstream docs variant

### PR DESCRIPTION
**Pull request by jistr**
Downstream docs preview can be built with:

```
BUILD=downstream make docs-user
```

(and analogously for the `docs-user-open` and `docs-user-watch` make targets).

### CODE REVIEW

**Comment by jistr** [comment]
Here are the docs upstream/downstream builds for preview in a section where they differ (the image URLs):

https://logserver.rdoproject.org/86/286/f963006c259fc7179d0a8c134c55e52cb61d8ea3/github-check/adoption-docs-preview/2303821/docs_build/docs_build/adoption-user/index-upstream.html#_procedure_telemetry_adoption

https://logserver.rdoproject.org/86/286/f963006c259fc7179d0a8c134c55e52cb61d8ea3/github-check/adoption-docs-preview/2303821/docs_build/docs_build/adoption-user/index-downstream.html#_procedure_telemetry_adoption

File: docs_user/modules/openstack-autoscaling_adoption.adoc [hunk 1]
```diff
@@ -34,17 +34,26 @@ spec:
         [DEFAULT]
         debug=true
       secret: osp-secret
+ifeval::["{build}" == "upstream"]
```
Comment (fao89) [inline docs_user/modules/openstack-autoscaling_adoption.adoc:37]: nice!

**Review by fao89 — APPROVED** [review]

**Comment by klgill** [comment]
I noticed that the rdoproject.org/github-check check failed, but otherwise, I approve.

**Review by klgill — APPROVED** [review]

**Comment by jistr** [comment]
recheck

**Comment by jistr** [comment]
recheck

**Comment by jistr** [comment]
recheck

**Comment by jistr** [comment]
recheck

**Pull request merged**